src/lib.rs

```diff
@@ -160,22 +183,61 @@ impl Drop for IRRootSignature {
     }
 }
 
+#[derive(Error, Debug)]
+pub enum RootSignatureError {
+    #[error("Failed to create IRRootSignature: {0:?}")]
+    CreateError(ffi::IRErrorCode),
```

Comment:
Didn't we plan on having a pretty-printer for `IRErrorCode`? I think this prints the (rather meaningless) number now doesn't it?

Reply:
It prints: IRObjectCompileError(CompilationError)

Reply:
Ah cool because it's a `--rustified-enum`, not a `--newtype-enum`.